### PR TITLE
LPS-123707 Add rich_text.ftl

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Review by jeyvison — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/render/DDMFormFieldFreeMarkerRenderer.java [hunk 1]
```diff
@@ -750,6 +736,34 @@ private Locale _getPreferredLocale(
 		return iterator.next();
 	}
 
+	private TemplateResource _updateTemplateResource(
+			String fieldNamespace, String templateName,
+			TemplateResource templateResource)
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/render/DDMFormFieldFreeMarkerRenderer.java:740]: You're passing templateResource but where are you using it?

Also can this method be inline?

**Review by jeyvison — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/render/DDMFormFieldFreeMarkerRenderer.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/render/DDMFormFieldFreeMarkerRenderer.java:740]: Inline -> changed.

templateResource has the default template (text.ftl) so can be returned if the conditions don't work.

**Comment by marcelabc** [comment]
@jeyvison could you please review again? I pushed some changes 👍

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
resent. See https://github.com/liferay-data-engine/liferay-portal/pull/51

**Pull request closed without merging**